packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -75,7 +75,7 @@ Posts.addView("userPosts", terms => ({
   options: {
     limit: 5,
     sort: {
-      score: -1,
+      postedAt: -1,
```

Comment:
Needs a corresponding index change (to the index right below).

Reply:
I believe is fixed now